docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,629 @@
+---
+layout: vNext
+title:  "Setting up and Running a Performance Benchmark on an existing network"
+categories: 
+permalink: /vNext/fabric-tutorial/tutorials-fabric-existing/
+order: 
+---
+## Table of Contents
+{:.no_toc}
+
+- TOC
+{:toc}
+
+## Overview
+This tutorial takes you through performance testing a smart contract on a pre-existing Fabric network using Caliper. 
+
+To complete this tutorial you will need to have installed NodeJS. To do this, we recommend using [nvm](https://github.com/nvm-sh/nvm).
+
+This tutorial follows from the test-network [fabric tutorial](https://hyperledger-fabric.readthedocs.io/en/release-2.2/tutorials.html). You need to have installed and instantiated the javascript `asset-transfer-basic` smart contract using the Fabric tutorial command `./network.sh deployCC -ccn basic -ccl javascript` (part two of the Fabric tutorial).
+
+> The following command list is a minimalist quick step guide to get the Fabric network up and running. __To understand and troubleshoot them, refer to the Fabric documentation linked above!__
+
+```bash
+# Clone a fixed version of the Hyperledger Fabric samples repo
+git clone https://github.com/hyperledger/fabric-samples.git
```

Comment:
Unfortunately these instructions aren't correct. I think they should be removed and we should stick to what is described in the fabric tutorial itself with the option to explain how to install the node version of the basic chaincode rather than the Go version as a suggestion before pointing to the tutorial

Reply:
1. True, the checkout needs a newer tag/commit, but we should choose a commit, so the tutorial won't break after some changes in the samples (current master is `22393b629bcac7f7807cc6998aa44e06ecc77426`). 
2. The curl line could also clone the repo, but since we want a specific version, I extracted the cloning part. 
3. The last lines are from the Fabric tutorial, but since it's a bit wordy, I extracted the gist, so users can focus on the Caliper part. 
4. We should reference the `Installing and Running Caliper/Pre-requisites` doc section for Caliper deps.